shardingsphere-features/shardingsphere-encrypt/shardingsphere-encrypt-core/src/main/java/org/apache/shardingsphere/encrypt/rule/EncryptRule.java

```diff
@@ -40,21 +40,21 @@
 import java.sql.SQLException;
 import java.util.Collection;
 import java.util.Collections;
-import java.util.LinkedHashMap;
 import java.util.LinkedList;
 import java.util.List;
 import java.util.Map;
 import java.util.Optional;
+import java.util.TreeMap;
 
 /**
  * Encrypt rule.
  */
 public final class EncryptRule implements SchemaRule, TableContainedRule {
     
     @SuppressWarnings("rawtypes")
-    private final Map<String, EncryptAlgorithm> encryptors = new LinkedHashMap<>();
```

Comment:
@cheese8 Can we use toLowerCase for map key instead of TreeMap?

Reply:
ok